src/node.cc

```diff
@@ -848,6 +849,26 @@ int InitializeNodeWithArgs(std::vector<std::string>* argv,
   if (per_process::cli_options->icu_data_dir.empty())
     credentials::SafeGetenv("NODE_ICU_DATA",
                             &per_process::cli_options->icu_data_dir);
+
+  #ifdef NODE_ICU_DEFAULT_DATA_DIR
+  // If neither the CLI option nor the environment variable was specified,
+  // fall back to the configured default
+  if (per_process::cli_options->icu_data_dir.empty()) {
+    // Check whether the NODE_ICU_DEFAULT_DATA_DIR contains the right data
+    // file and can be read.
+    char full_path[2048];
+
+    strcpy(full_path,
+           NODE_ICU_DEFAULT_DATA_DIR U_FILE_SEP_STRING NODE_ICU_DATA_FILE);
```

Comment:
Don't use strcpy() here since it's a static string, do this instead:
```c++
static const char full_path[] = NODE_ICU_DEFAULT_DATA_DIR "/" NODE_ICU_DATA_FILE;
```
(I don't think `U_FILE_SEP_STRING` is necessary since `/` is always a valid path separator on all platforms we support.)